README.md

```diff
@@ -23,10 +23,44 @@
 
 <p><img src="schedule.png" alt="LTS Schedule"/></p>
 
-The LTS Schedule is available also as a [JSON][] file or [ICal][]. There is
+The Release schedule is available also as a [JSON][] file or [ICal][]. There is
 also a live [Google Calendar][] that may be subscribed to.
 
-# LTS Plan
+## Mandate
+
+The Release working group's purpose is:
+
+* Management/execution of the release process for all releases.
+
+Its responsibilities are:
+
+* Define the release process.
+* Define the content of releases.
+* Generate and create releases. 
+* Manage the LTS and Current branches including backporting changes to 
+  these branches.
+* Define the policy for what gets backported to release streams.
```

Comment:
I don't have a good suggestion here.  We introduce the generic responsibilities, then the teams, and then the specific things the teams are entrusted with.  I'm not sure ordering differently would help.